ds2_configure.py

```diff
@@ -1126,6 +1128,16 @@ def additional_post_configurations():
         process = subprocess.Popen(shlex.split(command), stderr=subprocess.PIPE, stdout=subprocess.PIPE, shell=True)
         read = process.communicate()
         logger.info('base64postscript response: %s\n%s' % read)
+    if options.postscript_url:
+        try:
+            response = urllib.urlopen(options.postscript_url)
+            process = subprocess.Popen(response.read(), stderr=subprocess.PIPE, stdout=subprocess.PIPE, shell=True)
+            read = process.communicate()
+            logger.info('postscript_url response: %s\n%s' % read)
```

Comment:
Is this going to log the whole shellscript? Make it debug or eliminate it if so.